cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I think the following would be nice to have in here as a cypress config:

```json
{
  "baseUrl": "http://localhost:8080",
  "watchForFileChanges": false,
  "screenshotOnRunFailure": false,
  "video": false,
  "viewportHeight": 900,
  "viewportWidth": 1280
}
```

Reply:
I've added this into my To Do notes once I start writing out proper tests :)

Reply:
Yep, that makes sense! 🚀